packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -27,16 +27,17 @@ import {
   statusDescriptionTagName,
 } from './transform';
 import { OT_REQUEST_HEADER } from './utils';
+import { SERVICE_RESOURCE } from '@opentelemetry/resources';
 /**
  * Zipkin Exporter
  */
 export class ZipkinExporter implements SpanExporter {
   static readonly DEFAULT_URL = 'http://localhost:9411/api/v2/spans';
   private readonly _logger: api.Logger;
-  private readonly _serviceName: string;
   private readonly _statusCodeTagName: string;
   private readonly _statusDescriptionTagName: string;
   private readonly _reqOpts: http.RequestOptions;
+  private _serviceName: string;
   private _isShutdown: boolean;
 
   constructor(config: zipkinTypes.ExporterConfig) {
```

Comment:
I believe serviceName is still a required attribute in the exporter config type

Reply:
You still have to make the service name optional in the config type.

Reply:
Do you mean in the `types.ts` file?
Because I have changed it here: https://github.com/rezakrimi/opentelemetry-js/blob/c0b56d37025dcfa9cb4caae7a54684c80661b657/packages/opentelemetry-exporter-zipkin/src/types.ts#L24
and the config object is now optional.

Reply:
I must have just missed it